README.md

```diff
@@ -149,6 +149,7 @@ Then, update the projects to include this additional file:
 * [CI Infrastructure](https://github.com/SonarSource/re-ci-infrastructure)
 * [CI Infrastructure docs](https://github.com/SonarSource/re-ci-infrastructure/blob/master/languages-dotnet/README.md)
 * [VM Images repository](https://github.com/SonarSource/re-ci-images)
+* [Provisioning repository](https://github.com/SonarSource/dotnet-infra/blob/main/src/dotnet_infra/constructs/asg.py)

```

Comment:
I chose to link the specific file instead of the repo root, since `asg.py` is a bit cryptic and having to search the repo for it is a bit painful.